### DISCUSSION TITLE
Automatically load next diff when previous is marked as viewed

### DISCUSSION THREAD

**Discussion by hughpv** [community/community#28904] [Pull Requests]
I have a PR open with over 1,000 files changed. Understandably, the PR viewer stops loading diffs beyond a certain point.

So I find myself going through a cycle:

* Click "Load diff"
* Review changes
* Mark as viewed
* Repeat

It would be awesome if the "Mark as viewed" action also triggered a load of the next diff if it's not already rendered.

P.S. here's a plug for #10197 while I'm at it. I would love it if I could just knock out the whole PR by tapping `J` over and over again.

**Comment by oleg-erom** [comment]
Or add a button "Load next 10 files"

![Screenshot_20220815_121014](https://user-images.githubusercontent.com/105630366/184609582-12a2916b-8c15-4b56-93d6-81cad51b4ef6.jpg)

**Comment by david-kariuki** [comment]
Honestly this would be a good addition. Clicking Load Diff on a PR with hundreds of files is hectic
